x-pack/test/case_api_integration/security_and_spaces/tests/common/cases/post_case.ts

```diff
@@ -36,68 +40,181 @@ export default ({ getService }: FtrProviderContext): void => {
+              name: 'Jira',
+              type: ConnectorTypes.jira,
+              fields: { issueType: 'Task', priority: 'High', parent: null },
+            },
+          })
+        );
+        const data = removeServerGeneratedPropertiesFromCase(postedCase);
 
-      const data = removeServerGeneratedPropertiesFromCase(postedCase);
-      expect(data).to.eql(postCaseResp(postedCase.id));
-    });
+        expect(data).to.eql(
+          postCaseResp(
+            null,
+            getPostCaseRequest({
+              connector: {
+                id: '123',
+                name: 'Jira',
+                type: ConnectorTypes.jira,
+                fields: { issueType: 'Task', priority: 'High', parent: null },
+              },
+            })
+          )
+        );
+      });
 
-    it('unhappy path - 400s when bad query supplied', async () => {
-      await supertest
-        .post(CASES_URL)
-        .set('kbn-xsrf', 'true')
-        // @ts-expect-error
-        .send({ ...getPostCaseRequest({ badKey: true }) })
-        .expect(400);
-    });
+      it('should post a case: none connector', async () => {
+        const postedCase = await createCase(
+          supertest,
+          getPostCaseRequest({
+            connector: {
+              id: 'none',
+              name: 'none',
+              type: ConnectorTypes.none,
+              fields: null,
+            },
+          })
+        );
+        const data = removeServerGeneratedPropertiesFromCase(postedCase);
+
+        expect(data).to.eql(
+          postCaseResp(
+            null,
+            getPostCaseRequest({
+              connector: {
+                id: 'none',
+                name: 'none',
+                type: ConnectorTypes.none,
+                fields: null,
+              },
+            })
+          )
+        );
+      });
 
-    it('unhappy path - 400s when connector is not supplied', async () => {
-      const { connector, ...caseWithoutConnector } = getPostCaseRequest();
+      it('should create a user action when creating a case', async () => {
+        const postedCase = await createCase(supertest, getPostCaseRequest());
+        const userActions = await getAllUserAction(supertest, postedCase.id);
+        const creationUserAction = removeServerGeneratedPropertiesFromUserAction(userActions[0]);
 
-      await supertest
-        .post(CASES_URL)
-        .set('kbn-xsrf', 'true')
-        .send(caseWithoutConnector)
-        .expect(400);
-    });
+        expect(creationUserAction).to.eql({
+          action_field: ['description', 'status', 'tags', 'title', 'connector', 'settings'],
+          action: 'create',
+          action_by: defaultUser,
+          new_value: `{"type":"${postedCase.type}","description":"${
+            postedCase.description
+          }","title":"${postedCase.title}","tags":${JSON.stringify(
+            postedCase.tags
+          )},"connector":${JSON.stringify(postedCase.connector)},"settings":${JSON.stringify(
```

Comment:
Does `JSON.stringify` produce a consistent order of values? We might need to sort the keys or array or something to ensure that this test isn't flaky?

Reply:
Good catch! According to [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) the order is not guaranteed.

> Note: Properties of non-array objects are not guaranteed to be stringified in any particular order. Do not rely on ordering of properties within the same object within the stringification.

Nevertheless, neither the insertion order in ES is guaranteed because we `JSON.stringify` the new value when creating the user action. 

I will change all of these checks accordingly.